packages/auth/src/grant/service.test.ts

```diff
@@ -135,38 +135,46 @@ describe('Grant Service', (): void => {
         type: AccessType.IncomingPayment
       })
     })
-    test('Can issue a grant without interaction', async (): Promise<void> => {
-      const grantRequest: GrantRequest = {
-        ...BASE_GRANT_REQUEST,
-        access_token: {
-          access: [
-            {
-              ...BASE_GRANT_ACCESS,
-              type: AccessType.IncomingPayment
-            }
-          ]
-        },
-        interact: undefined
-      }
+    test.each`
+      type                          | state                  | interact                       | not       | description
+      ${AccessType.IncomingPayment} | ${GrantState.Approved} | ${undefined}                   | ${''}     | ${'incoming payment'}
+      ${AccessType.Quote}           | ${GrantState.Approved} | ${undefined}                   | ${''}     | ${'quote'}
+      ${AccessType.OutgoingPayment} | ${GrantState.Pending}  | ${BASE_GRANT_REQUEST.interact} | ${' not'} | ${'outgoing payment'}
+    `(
+      'Does$not approve a grant without interaction - $description',
```

Comment:
Nit-picky, but instead of using `$not` we can just put everything in the description

Reply:
I tried but I couldn't come up with something that made sense for all scenarios